apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -661,12 +667,19 @@ function initializeBlocklyWrapper(blocklyInstance) {
     const trashcan = new CdoTrashcan(workspace);
     trashcan.init();
 
+    blocklyWrapper.setMainWorkspace(workspace);
+
+    // Hidden workspace where we can put function definitions.
```

Comment:
You could add to comment this is where we also put 'hidden' behavior definitions.

Reply:
I'm going to not refer to behaviors for now since we have not implemented that work yet, and things might change.